WyattHWilliams/react-challenge-project-jan-2022
# #7 Update Dependencies

## Changes
1. updated package files
2. refactored appRouter for react-router-dom v6+
3. refactored login component for react-router-dom v6+

## Purpose
to update dependencies and refactor based on dependency changes.
addresses [issue 7](https://github.com/Shift3/react-challenge-project-jan-2022/issues/7)

## Approach
1. usage of BrowserRouter in react-router-dom v6 has changed
2. useHistory is no longer in react-router-dom v6
3. route components in the element prop don't recieve router-props any more. (i.e. this.history.push('/') is no longer possible)

## Testing Steps
1. navigate to each route on the app to ensure the router is working correctly.

Closes #007
